cfe_internal/enterprise/federation/federation.cf

```diff
@@ -1,6 +1,6 @@
 body file control
 {
-        inputs => { "$(sys.libdir)/stdlib.cf" };
+        inputs => { "$(sys.libdir)/stdlib.cf", "cfe_internal/enterprise/CFE_hub_specific.cf" };
```

Comment:
With this input path, what happens if you try to run the federation policy directly leveraging bundle agent __main__?

Reply:
Good catch. I added `$(sys.inputdir)` there.